### DISCUSSION TITLE
Use of libMXF and libMXF++

### DISCUSSION THREAD

**Discussion by nishabfhg** [bbc/bmx#4] [Q&A] [answered]
Hi, 

We would like to use your libMXF and libMXF++ libraries within some applications built around the codec, JPEG XS. I would like to know:
1. if these libraries are still being maintained and how long they are planned to be supported. 
2. in case we are able to successfully integrate your libraries, would it be possible for us to add JPEG XS to your list of supported codecs? 

Thank you, 
Nisha

**Comment by philipnbbc** [comment, marked as answer]
Thanks for your interest Nisha. The libraries, libMXF, libMXF++ and bmx are being maintained and updates made as required. There is no software support as such provided for the libraries and so no planned support duration. We are happy for you to contribute code that adds JPEG XS (I guess based on SMPTE ST 2124) and in that case JPEG XS would be added to the list of supported formats.

**Reply by nishabfhg** [reply]
Hi Philip, Thanks for your reply. Yes, it will be based on SMPTE ST 2124. And what would be the best way to contribute to your repo? Should I create a fork?

**Reply by philipnbbc** [reply]
Yes, create a fork and then a pull request when ready for review would work well, thanks.
